packages/esm-sms-app/src/providers/test-form/provider-config-test-form.test.tsx

```diff
@@ -36,12 +35,10 @@ jest.mock('@openmrs/esm-framework', () => ({
 }));
 
 jest.mock('../../hooks/useLogs', () => ({
-  useLogsRecords: jest.fn(() => ({ mutateLogs: jest.fn() })),
+  useSMSLogs: jest.fn(() => ({ mutateLogs: jest.fn() })),
```

Comment:
The proper camel case will be `useSmsLogs`

Reply:
corrected